lib/cli/index.js

```diff
@@ -96,9 +97,17 @@ function runGemini(method, paths, options) {
         return gemini;
     })
         .then((gemini) => {
+            function parseReporterOptions(options) {
+                return options.reporter.map(function(name) {
```

Comment:
this code will fail when `options.reporter` is `undefined`, for example, when running `gemini update`